R/Record.R

```diff
@@ -148,43 +148,57 @@ Record <- R6::R6Class( # nolint object_name_linter
     .api = NULL,
     .data = NULL,
     get_value = function(key) {
```

Comment:
I changed the ordering of this function because I was starting to find it difficult to read

Reply:
👍🏻 I think I just didn't want to mess with the existing structure but this is better.